src/test/java/seedu/address/testutil/EditPersonDescriptorBuilder.java

```diff
@@ -55,6 +68,14 @@ public EditPersonDescriptorBuilder withPhone(String phone) {
         return this;
     }
 
+    /**
+     * Add a duplicate phone field to the {@code EditPersonDescriptor} that we are building.
+     */
+    public EditPersonDescriptorBuilder withDuplicatePhone() {
+        descriptor.addDuplicateField(PREFIX_PHONE.getPrefix());
+        return this;
+    }
```

Comment:
It's possible but it might be less intuitive to when building the test cases since calling the same method twice may not be that intuitive as compared to calling it via a specific method.

I will try to experiment combining the functionality in the same method.

Reply:
I felt calling the method `withPhone()` twice is the more natural way to specify the same parameter twice. With a specific method like `withDuplicatePhone()`, you open the door for calling `withDuplicatePhone()` without calling `withPhone()` first.
Yes, experiment variations and see how it goes.